tests/csrf_tests/tests.py

```diff
@@ -185,10 +185,21 @@ def test_csrf_token_in_header(self):
         Check that we can pass in the token in a header instead of in the form
         """
         req = self._get_POST_csrf_cookie_request()
-        req.META['HTTP_X_CSRFTOKEN'] = self._csrf_id
+        req.META[settings.CSRF_HEADER_NAME] = self._csrf_id
```

Comment:
That's a negligible detail anyway, but here's the reasoning behind my comment, for your information.

Users of Django must write JS code to inject a header called X-CSRFToken. Changing the default value of the setting is backwards incompatible because users will have to change their JS code. It would be nice if tests failed in that case.

The general principle at play here is that factoring tests shrinks them until they're a copy of the code they're testing, at which point they don't have any value. It's more important for tests to be explicit than DRY.